webapp/channels/src/components/admin_console/password_settings.tsx

```diff
@@ -184,7 +227,7 @@ export default class PasswordSettings extends AdminSettings {
                             <label className='checkbox-inline'>
                                 <input
                                     type='checkbox'
-                                    ref={this.lowercase}
+                                    ref={this.Lowercase}
```

Comment:
where are these defined? do we even need these refs?

Reply:
I defined them just above the class constructor. From line 55 to line 58. i wasn't quite sure if we needed them or not so i decided to leave them in.